### PR TITLE
Remove EnvironmentUtil in the cluster module and optimize UT

### PR DESCRIPTION
**Pull request by jt2594838**
Replace EnvironmentUtil in the cluster module with the one in the server module.
Reduce deregister timeout and disable WAL in tests to accelerate tests.
Fix query context leak in some tests.

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/query/control/QueryFileManager.java [hunk 1]
```diff
@@ -119,6 +121,9 @@ void addFilePathToMap(long queryId, TsFileResource tsFile, boolean isClosed) {
     if (!pathMap.get(queryId).contains(tsFile)) {
       pathMap.get(queryId).add(tsFile);
       FileReaderManager.getInstance().increaseFileReaderReference(tsFile, isClosed);
+      logger.warn("{} is read-locked by {}", tsFile, queryId, new Exception());
     }
   }
+
+  private static final Logger logger = LoggerFactory.getLogger(QueryFileManager.class);
```
Comment (mychaow) [inline server/src/main/java/org/apache/iotdb/db/query/control/QueryFileManager.java:128]: may put this to the first line of Class will be better.

**Review by mychaow — APPROVED** [review]
Good job!

File: cluster/src/test/java/org/apache/iotdb/cluster/server/member/MemberTest.java [hunk 2]
```diff
@@ -149,6 +156,7 @@ public void setUp() throws Exception {
 
     testMetaMember.setPartitionTable(partitionTable);
     MetaPuller.getInstance().init(testMetaMember);
+    logger.warn("Member test set up");
```
Comment (neuyilan) [inline cluster/src/test/java/org/apache/iotdb/cluster/server/member/MemberTest.java:159]: Why use the warn log here?

File: server/src/main/java/org/apache/iotdb/db/query/control/QueryFileManager.java [hunk 3]
```diff
@@ -119,6 +121,9 @@ void addFilePathToMap(long queryId, TsFileResource tsFile, boolean isClosed) {
     if (!pathMap.get(queryId).contains(tsFile)) {
       pathMap.get(queryId).add(tsFile);
       FileReaderManager.getInstance().increaseFileReaderReference(tsFile, isClosed);
+      logger.warn("{} is read-locked by {}", tsFile, queryId, new Exception());
```
Comment (neuyilan) [inline server/src/main/java/org/apache/iotdb/db/query/control/QueryFileManager.java:124]: Is this a debug log? Why print the exception?

**Review by neuyilan — APPROVED** [review]

File: server/src/main/java/org/apache/iotdb/db/query/control/QueryFileManager.java (hunk 1 above)
Comment (jt2594838) [reply server/src/main/java/org/apache/iotdb/db/query/control/QueryFileManager.java:128]: I added it just to print some logs in tests for debugging, and I will remove it.

File: server/src/main/java/org/apache/iotdb/db/query/control/QueryFileManager.java (hunk 3 above)
Comment (jt2594838) [reply server/src/main/java/org/apache/iotdb/db/query/control/QueryFileManager.java:124]: Removed.

File: cluster/src/test/java/org/apache/iotdb/cluster/server/member/MemberTest.java (hunk 2 above)
Comment (jt2594838) [reply cluster/src/test/java/org/apache/iotdb/cluster/server/member/MemberTest.java:159]: It was for debugging, so I will remove it.

File: cluster/pom.xml [hunk 4]
```diff
@@ -77,6 +77,13 @@
             </exclusions>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.iotdb</groupId>
+            <artifactId>iotdb-server</artifactId>
+            <version>0.12.0-SNAPSHOT</version>
```
Comment (jixuan1989) [inline cluster/pom.xml:83]: using ${project.version} to keep consistent?

**Pull request merged**